src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random RANDOM = new Random();
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
+    private static final int MAGIC_NUMBER = 100;
```

Comment:
give self descriptive name
magic number is a term, pls google it to know
```suggestion
    private static final int MAGIC_NUMBER = 100;
```